genestack_client/share_util.py

```diff
@@ -76,7 +78,9 @@ def share_files_for_edit(self, file_accessions, group_accession, destination_fol
                will be created if this parameter is equal to `None`.
         :type destination_folder: str
         """
-        self.__share(file_accessions, group_accession, destination_folder, 'shareFilesForEditing')
+        self.safe_share_files(
+            file_accessions, group_accession, [ShareUtil.Permissions.EDIT], destination_folder
```

Comment:
Do we need to add view permission here?

Reply:
No, these permissions are not the same as in the core, so EDIT includes VIEW as well